lib/ansible_tower_client/inventory_source.rb

```diff
@@ -0,0 +1,21 @@
+module AnsibleTowerClient
+  class InventorySource < BaseModel
+    def self.endpoint
+      "inventory_sources".freeze
+    end
+
+    def can_update?
```

Comment:
Since the field in the Tower API is "can_update", we should probably leave it as you wrote it.